tests/system/tio.nim

```diff
@@ -1,6 +1,6 @@
 discard """
 outputsub: ""
-disabled: true
+    disabled: true
```

Comment:
Disabled around 2011-12 because of infinite blocking, reenabled shortly after
References #7121
Disabled around 2018-11 due to 'just causing problems'

Reply:
These ii tests really half-baked.

Exterminate!